bundle/manifests/tech-preview/support-log-gather-operator.clusterserviceversion.yaml

```diff
@@ -208,6 +208,8 @@ spec:
                   value: must-gather-operator
                 - name: DEFAULT_MUST_GATHER_IMAGE
                   value: quay.io/openshift/origin-must-gather:latest
+                - name: ACM_HCP_MUST_GATHER_IMAGE
+                  value: registry.redhat.io/multicluster-engine/must-gather-rhel9:v2.8
```

Comment:
I don't know a ton about this image - should this be pointing at quay.io for OCP deployments?

@TrilokGeer @vaidehi411 @swghosh @shivprakashmuley - any thoughts here? Would love to lean on your expertise.

Reply:
These are typically placeholder images, which get replaced with acceptable production grade images (i.e. generally registry.redhat.io ones) once building through our ART pipelines.